### PR TITLE
Impl command to compute base64 contract hashes

### PR DESCRIPTION
**Pull request by kettlebell**
For verification against EIP-23.

### CODE REVIEW

**Review by greenhat — CHANGES_REQUESTED** [review]
The separate command is a great idea! Thank you! 
Could you please add a unit test as well? It'll give us a check on contract updates in PRs.
Plus, it'll work for those who don't want to download and run binaries.

**Comment by kettlebell** [comment]
That's a good idea! It's done.

File: core/src/default_parameters.rs [hunk 1]
```diff
@@ -158,3 +158,11 @@ pub fn print_contract_hashes() {
         encoded_hash(update_ergo_tree_bytes)
     );
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn print_contract_hashes() {
+        super::print_contract_hashes();
```
Comment (greenhat) [inline core/src/default_parameters.rs:166]: I was thinking of hard-coding the hashes in the unit tests.
The workflow I have in mind for checking a contract's hash in the unit test is the following:
- Let's say we have a PR that changes a contract (P2S string in defaults);
- On PR review, I want to make sure that the updated contract is the same contract that is defined in EIP (or somewhere else); 
- I check that the updated hash in the unit test is the same hash as in EIP(or somewhere else);

**Review by greenhat — CHANGES_REQUESTED** [review]

File: core/src/default_parameters.rs (hunk 1 above)
Comment (kettlebell) [reply core/src/default_parameters.rs]: Sounds good, I've added the hashes in.

**Review by greenhat — APPROVED** [review]
Great! Thank you!

**Pull request merged**